### PR TITLE
adding notes from infrastructure working group (2022-2024)

### PR DESCRIPTION
**Pull request by aleesteele**
Hi @the-turing-way/infrastructure! Here is the archive of our notes from HackMD: https://hackmd.io/@turingway/infra-wg-notes

Take a look and see if there's anything missing. It's great to have this public archive out there!

### CODE REVIEW

**Review by JimMadge — APPROVED** [review]
LGTM :+1:

File: meeting-notes/20231207-notes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- Sarah
+- Ale
+- Brigitta
+
+
+Notes:
+- Help Angus to understand the sense of this meeting
+- What we would like to accomplish 
+- Angus backgroung: using jupyterhub for some time. Experience with Onboarding people ---> Jupyterhub has a barrier if people are not technical 
+    - Identify core offerings from Jupyterhub
```
Comment (sgibson91) [inline meeting-notes/20231207-notes.md]: I think all mentions of JupyterHub here are actually meant to be Jupyter _Book_ as that is the project Angus is working on

File: meeting-notes/20231207-notes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - How can we onboard new users into the new Jupiterhub notebook ---> figure out if there are opportunities of new features that could be beneficial for the community
```
Comment (sgibson91) [inline meeting-notes/20231207-notes.md]:
```suggestion
    - How can we onboard new users into the new Jupyter book ---> figure out if there are opportunities of new features that could be beneficial for the community
```

**Review by sgibson91 — APPROVED** [review]

File: meeting-notes/20231207-notes.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- Angus
+- Sarah
+- Ale
+- Brigitta
+
+
+Notes:
+- Help Angus to understand the sense of this meeting
+- What we would like to accomplish 
+- Angus backgroung: using jupyterhub for some time. Experience with Onboarding people ---> Jupyterhub has a barrier if people are not technical
```
Comment (aleesteele) [inline meeting-notes/20231207-notes.md]:
```suggestion
- Angus backgroung: using Jupyter Book for some time. Experience with Onboarding people ---> Jupyter Book has a barrier if people are not technical 
```

File: meeting-notes/20231207-notes.md (hunk 1 above)
Comment (aleesteele) [reply meeting-notes/20231207-notes.md]: Oh yes, I can see what you mean! Thanks @sgibson91 - did I get all the corrections here?

**Pull request merged**